snapshots/wb/2025-06-05/pip_api.py

```diff
@@ -431,6 +436,13 @@ def pip_query_country(
     version = versions[ppp_version]["version"]
     release_version = versions[ppp_version]["release_version"]
 
+    # NOTE: There is a bug for China: when querying specific poverty lines (as for relative poverty), the API doesn't respond
+    # One hacky way to fix it is to call a more generalized query first (with no welfare_type, reporting_level nor release version)
+    if country_code == "CHN":
```

Comment:
@Marigold I think I've found a better solution for this issue of the API not working for some (correct) queries. Weirdly, it works after querying data for the country in a more generalized query, so I am doing that now for China (which seems to be the only offender)